src/website/views/components/alert.pug

```diff
@@ -16,76 +17,153 @@ h4 Base
-.example.-mb--3
-  .-p--1
-    .m-alert.-bubble.-warning.-hasIcon.-m--2(data-title="Warning")
-      | This is a warning alert
+p.-text Use the class <code>-dismiss</code> and a button to allow the user to dismiss the alert.
+.example.-mb3
+  .-p1
+    .m-alert.-bubble.-dismiss.-danger.-m--2
+      .m-alert__content
+        i.a-icon.-sm2.icon-circle-warning.-text--danger
+        | This is a danger alert
+      button(class='a-btn -icon -close -small')
+        .a-btn__content
+          i.a-icon.icon-x
+    .m-alert.-bubble.-large.-dismiss.-danger.-m--2
+      .m-alert__content
+        i.a-icon.-sm2.icon-circle-warning.-text--danger
+        div
+          h5.a-h5 danger
+          | This is a danger alert
+      button(class='a-btn -icon -close -small')
+        .a-btn__content
+          i.a-icon.icon-x
+    .m-alert.-banner.-dismiss.-danger.-m--2
+      .m-alert__content
+        i.a-icon.-sm2(class ='icon-circle-warning -text--danger')
+        | This is a danger alert
       button(class='a-btn -icon -close -small')
         .a-btn__content
           i.a-icon.icon-x
   :code(lang="html")
-      <div class="m-alert -warning -hasIcon" data-title="Warning" role="alert">
-        This is a warning alert
-        <button class="a-btn -icon -close -small">
-          <div class="a-btn__content">
-            <i class="a-icon icon-x"></i>
-          </div>
-        </button>
+    <div class="m-alert -bubble -dismiss -warning">
+      <div class="m-alert__content">
+        <i class="a-icon -sm2 icon-circle-warning -text--warning"></i>
+        This is a warning alert<button class="a-btn -icon -close -small">
+      </div>
+      <button class="a-btn -icon -close -small">
+        <div class="a-btn__content">
+          <i class="a-icon icon-x"></i>
+        </div>
+      </button>
+    </div>
+    <div class="m-alert -bubble -large -dismiss -danger">
+      <div class="m-alert__content">
+        <i class="a-icon -sm2 icon-circle-warning -text--danger"></i>
+        <div>
+          <h5 class="a-h5">Danger</h5>
+          This is a danger alert
+        </div>
+      </div>
+      <button class="a-btn -icon -close -small">
+        <div class="a-btn__content">
+          <i class="a-icon icon-x"></i>
+        </div>
+      </button>
+    </div>
+    <div class="m-alert -banner -dismiss -danger">
+      <div class="m-alert__content">
+        <i class="a-icon -sm2 icon-circle-warning -text--danger"></i>
+        This is a danger alert
       </div>
+      <button class="a-btn -icon -close -small">
+        <div class="a-btn__content">
+          <i class="a-icon icon-x"></i>
+        </div>
+      </button>
+    </div>
 
 h4 Sizes
 h5 Large
-p.-text Apply a <code>-large</code> class to increase the size of the alert.
-.example.-mb--3
-  .-p--1
-    .m-alert.-bubble.-warning.-large.-hasIcon.-m--2(data-title="Warning")
+p.-text
+  | Apply a <code>-large</code> class to increase the size of the alert and
```

Comment:
@davidojedacl @jllr Text abruptly cuts off here. Can we revert to the previous copy? `Apply a <code>-large</code> class to increase the size of the alert.`

Reply:
fixed

Reply:
fixed